checker/src/org/checkerframework/checker/initialization/InitializationAnnotatedTypeFactory.java

```diff
@@ -575,8 +591,27 @@ private void computeFieldAccessType(AnnotatedTypeMirror type,
                     .asType();
             boolean isInitializedForFrame = isInitializedForFrame(receiverType, fieldDeclarationType);
             if (isInitializedForFrame) {
+                // receiver is initialized for this frame.
+                // change the type of the field to @UnknownInitialization so that
+                // anything can be assigned to this field.
                 type.replaceAnnotation(qualHierarchy.getTopAnnotation(UNCLASSIFIED));
+
+            } else if(computingAnnotatedTypeMirrorOfLHS) {
+                // receiver is not initialized for this frame and the type of a lhs is being computed
```

Comment:
Same comment about capitalization as above. Also full stop or comma at the end of this line?